pkg/catalog/routes.go

```diff
@@ -405,3 +405,33 @@ func (mc *MeshCatalog) listTrafficTargetPermutations(trafficTarget target.Traffi
 
 	return trafficPolicies, nil
 }
+
+// routesFromRules takes a set of traffic target rules and a namespace and returns a list of http routes (trafficpolicy.HTTPRoute)
+func (mc *MeshCatalog) routesFromRules(rules []target.TrafficTargetRule, namespace string) ([]trafficpolicy.HTTPRoute, error) {
```

Comment:
Can we describe what namespace refers to here, and also name the `namespace` var accordingly, ex. `trafficTargetNamespace`.